src/webpack/buildPreview.js

```diff
@@ -84,13 +64,12 @@ module.exports = async function buildPreview(result, outputDir) {
         };
       }, {})
 
-
       return [
         ...acc,
         {
           bundleName,
           ...dimensions,
-          maxFileSize: result[i]?.settings?.data?.settings?.maxFileSize,
+          maxFileSize: result && result[i].settings.data.settings.maxFileSize,
```

Comment:
Can you explain this a bit more? Now it checks if there is a result, but not if settings, data etc exist right?

Reply:
if there is a result - there are banners. And all banners have settings.data and data have settings. maxFileSize can be there or be undefined. If undefined - we use default 150 in react. I've added the first "result &&" to make sure I have something and not just trying to access an undefined "array" with [i].